### ISSUE TITLE
Mac OS 10.3.4

### ISSUE THREAD

**Issue by ddiazescandon** [w0lfschild/Flashlight#44]
I'm not sure if it is not working or I'm just a noob
SIP is off
MySIMBL is up and running
![screen shot 2018-04-01 at 19 31 13](https://user-images.githubusercontent.com/10009779/38179000-56ed5680-35e3-11e8-917d-08640ef88cc3.png)
![screen shot 2018-04-01 at 19 30 52](https://user-images.githubusercontent.com/10009779/38179010-72621130-35e3-11e8-87db-a93f560494c7.png)

But Flashlight says some packages are installed and none of it works, I mean,  I have seen previews  
installations and there was a toogle ON and OFF, and 1.0.8 doesn't have it  ( or it isn't showing up)
![screen shot 2018-04-01 at 19 31 32](https://user-images.githubusercontent.com/10009779/38179004-662f0d6e-35e3-11e8-8e09-aae1d8bef31f.png)

**Comment by dkgrieshammer** [comment]
Same for me, also 10.3.4

**Comment by Gudsfile** [comment]
> Same for me, also 10.3.4

Same for me

**Comment by w0lfschild** [comment]
Is the latest version working?

**Comment by dkgrieshammer** [comment]
Didn't try on Mojave yet; anybody tried it?

**Comment by waxfingers** [comment]
I'm pretty sure that thumbs up above means (at least) 3 confirmations. Just not to add more confusion: I'm **on 10.14.1 and it is working**; I have still not tested with SIP re-enabled; I'm also running XtraFinder. From MySimbl almost all the plugs I've tested do work but Fullscreener and GlobalNoWindoTitle... WinBuddy seems to have some issues too (after a while it freezes the system), this could be due to some incompatibilities with me running also BTT and Karabiner Elements. From Flashlight all the plugs I've tried do load, much the same but there are a handful of plugins that do not work, read "broken". As in any neighbourhood, there are applications with MySimbl & Falshlight incompatibilities, these are too specific to be listed her. Cheers
